fish.desktop

```diff
@@ -5,7 +5,8 @@ Terminal=true
 # This file is for use in appimages, which rewrite the path.
 # Other uses would have to replace these with the correct path.
 Exec=fish
-Name=fish
+Name=Fish
```

Comment:
I don't think we use uppercase fish anywhere

Reply:
Also looking for a reasoning for the capital F fish.

Reply:
It was just more good-looking (at least on Plasma)